infracheck/checks/smtp_credentials_check.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python3
+"""
+Verifies SMTP credentials by using smtplib module to log in at the server.
```

Comment:
Missing` <sphinx>` tags

Example:
https://github.com/riotkit-org/infracheck/blob/master/infracheck/checks/free-ram#L4

Reply:
And missing entry in https://raw.githubusercontent.com/riotkit-org/infracheck/master/docs/source/reference.rst :grin:

Reply:
And the last thing - the formatting should be RST

```
smtp_credentials_check.py
------------------------------------

Arguments are provided via environmental variables:

- smtp_host
- smtp_port
- smtp_user
- smtp_password
```

Lowercase is because the names are converted to uppercase later - in Infracheck application the user gives lowercase letters (the user does not care that those are environment variables)

Reply:
Effect will be there after merging: https://infracheck.docs.riotkit.org/en/latest/reference.html#free-ram

:grin:

Reply:
Sorry, the </sphinx> is also missing :joy: